source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst

```diff
@@ -670,6 +731,33 @@ Enabling camera sensors
         sink_1::startx="<640>" sink_1::starty="<480>" sink_1::widths="<640>" sink_1::heights="<480>" ! \
         queue ! kmssink driver-name=tidss plane-properties=s,zpos=1
 
+    CSI2RX testing details
+    ======================
+
+    Following sensors and daughter cards have been tested with the latest SDK.
+
+        +------------------------------------+----------------------------+----------------------+------------+
```

Comment:
Same recommendation of using csv-table as above

Reply:
Hello @cshilwant, I have done the requested changes, please review.